confgenerator/covprofile

```diff
@@ -0,0 +1,9433 @@
+mode: set
```

Comment:
This file doesn't need to be checked in does it?

Reply:
I have no idea how it got in there. Removing it now.